src/sidebar/store/test/util-test.js

```diff
@@ -27,7 +27,7 @@ const fixtures = {
       selectors: {
```

Comment:
The `useLocalState` flag is no longer used by the implementation, so the corresponding tests can be removed as well.